docs/known-issues.rst

```diff
@@ -110,3 +110,23 @@ For **Flatpak** application, see `this workaround
     for details on how to add the command).
 
     `Issue #27 <https://github.com/hluk/CopyQ/issues/27>`__
+
+Scripting command "copy()" fails
+--------------------------------
+
+The command ``copy()`` sends the Ctrl+C shortcut to the current window.
+This can fail depending on the active application.
+If CopyQ won't detect a clipboard change, it throws an exception.
+The execution then fails with the message ``Failed to copy to clipboard!``.
+
+An alternative under Windows is to use a Powershell script as a command:
+  .. code-block:: powershell
+
+    powershell:
+      Add-Type -AssemblyName System.Windows.Forms;
+      [System.Windows.Forms.SendKeys]::SendWait("^c");
+      Start-Sleep -Milliseconds 150;
+
+The delay is added to make sure the text is copied to the clipboard.
+
+Beware that necessary **permissions** for Powershell independent of CopyQ need to have been setup.
```

Comment:
Can you provide more details? Some external link or simple description how to set up the permissions.